src/core/georeferencing.h

```diff
@@ -696,25 +712,34 @@ constexpr unsigned int Georeferencing::scaleFactorPrecision()
 	return 6u;
 }
 
+inline
+constexpr unsigned int Georeferencing::scaleFactorMultiplier()
+{
+	return 1000000.0;
```

Comment:
`return 1E6;` might be better aligned to the next patch which adds `1E8` for grivation precision.

Reply:
I plan to make this change.